e2e/tests/admin/whats-new.test.ts

```diff
@@ -0,0 +1,263 @@
+            await mockChangelog(page, [ENTRIES.newFeatured]);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+            await whatsNewPage.banner.waitForBanner();
+
+            await expect(whatsNewPage.banner.container).toBeVisible();
+            await expect(whatsNewPage.banner.title).toHaveText('New Featured Update');
+            await expect(whatsNewPage.banner.excerpt).toHaveText('This is an exciting new feature');
+        });
+
+        test('does not show toast when there are no new entries', async ({page}) => {
+            await mockChangelog(page, [ENTRIES.old]);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+
+            await expect(whatsNewPage.banner.container).not.toBeVisible();
+        });
+
+        test('does not show toast when there are no entries at all', async ({page}) => {
+            await mockChangelog(page, []);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+
+            await expect(whatsNewPage.banner.container).not.toBeVisible();
+        });
+
+        test('does not show toast when latest entry is not featured', async ({page}) => {
+            await mockChangelog(page, [ENTRIES.newNonFeatured]);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+
+            await expect(whatsNewPage.banner.container).not.toBeVisible();
+        });
+
+        test('does not show toast for new users', async ({page}) => {
+            const userService = createUserService(page.request);
+            await userService.updateCurrentUserAccessibility({});
+
+            await mockChangelog(page, [ENTRIES.newFeatured]);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+
+            await expect(whatsNewPage.banner.container).not.toBeVisible();
+        });
+
+        test('does not show toast again after dismissal', async ({page}) => {
+            await mockChangelog(page, [ENTRIES.newFeatured]);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+            await whatsNewPage.banner.waitForBanner();
+
+            await expect(whatsNewPage.banner.container).toBeVisible();
+
+            await whatsNewPage.banner.dismiss();
+
+            await expect(whatsNewPage.banner.container).not.toBeVisible();
+
+            await whatsNewPage.goto();
+            await expect(whatsNewPage.banner.container).not.toBeVisible();
+        });
+
+        test('does not show toast again after clicking link', async ({page}) => {
+            await mockChangelog(page, [ENTRIES.newFeatured]);
+            const whatsNewPage = new WhatsNewPage(page);
+
+            await whatsNewPage.goto();
+            await whatsNewPage.banner.waitForBanner();
+
+            const [popup] = await Promise.all([
```

Comment:
wondering if we can hide this promise by encapsulating this into page object, with something like:

`whatsNewPage.waitForPopup`